keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -306,7 +312,7 @@ Additionally, privileged containers may impact other pod security policies (PSPs
    </td>
```

Comment:
For alpha, I recommend just committing to updating the pod security standards doc to disallow `hostProcess: true` in the baseline/default/restricted policies.

For the "open question if PSP should be updated", resolving that can be a requirement for graduating from alpha to beta. There are currently discussions about the shape of the PSP replacement that will make it clearer what needs to happen here.

Reply:
updated.